### DISCUSSION TITLE
Option to compare with previous bed level

### DISCUSSION THREAD

**Discussion by MatrixJson** [jneilliii/OctoPrint-BedLevelVisualizer#563] [Q&A] [answered]
I feel like I'm going crazy. I just had to re-install Octo-pi on my board and previously I was using this plugin but remember having a drop down of bed level data that I could switch between to compare the graphs. Since re-installing this plugin I'm not seeing the drop down and can't find any reference to this feature on the internet. Am I going crazy or was this a feature previously that was taken away?

**Comment by jneilliii** [comment, marked as answer]
You installed the unlisted [BLVLogger](https://github.com/jneilliii/OctoPrint-BLVLogger) companion plugin. Since it's not registered it doesn't get installed on restore.
